go/vt/vtgate/executor_test.go

```diff
@@ -833,6 +834,60 @@ func TestExecutorShow(t *testing.T) {
 			t.Errorf("%v:\n%+v, want\n%+v", query, qr, wantqr)
 		}
 	}
+	for _, query := range []string{"show charset like '%foo'", "show character set like 'foo%'", "show charset like 'foo%'","show character set where foo like 'utf8'", "show character set where charset like '%foo'", "show charset where charset = '%foo'"} {
+		qr, err := executor.Execute(context.Background(), "TestExecute", session, query, nil)
+		if err != nil {
+			t.Error(err)
+		}
+		wantqr := &sqltypes.Result{
+			Fields:       append(buildVarCharFields("Charset", "Description", "Default collation"), &querypb.Field{Name: "Maxlen", Type: sqltypes.Int32}),
+			Rows:         [][]sqltypes.Value{},
+			RowsAffected: 0,
+		}
+		if !reflect.DeepEqual(qr, wantqr) {
+			t.Errorf("%v:\n%+v, want\n%+v", query, qr, wantqr)
+		}
+	}
+	for _, query := range []string{"show charset like 'utf8'", "show character set like 'utf8'", "show charset where charset = 'utf8'", "show character set where charset = 'utf8'"} {
+		qr, err := executor.Execute(context.Background(), "TestExecute", session, query, nil)
+		if err != nil {
```

Comment:
There's a few of them in the new code, and it's probably because most of this file is doing it. 
I think it's way better to write `require.NoError(t, err)` here.